### PR TITLE
Docs: getting-started > Introduction

### PR DESCRIPTION
**Pull request by erondpowell**
TO DO:

- [x] Rewrite first draft
- [x] Submit for mgmt review
- [ ] Apply manager edit requests
- [ ] Add media
- [ ] Proof read
- [ ] Send to Lori for her reads
- [ ] Final Review
- [ ] Submit to Merge

### CODE REVIEW

File: docs/getting-started/introduction.md [hunk 1]
```diff
@@ -1,121 +1,223 @@
...
+-->
+
+**Directus is an Open Data Platform built to democratize the database.**
+
+This platform can manage any data, or data model, in any SQL database. Directus uses a technique called
+[Database Introspection](#how-it-works), which identifies your full database structure and creates an abstraction layer,
+then dynamically generates REST and GraphQL API endpoints to manage the data with exhaustive, granular control. These
+APIs are available for devs, but also used to power the intuitive no-code app.
+
+<!--Anything that the APIs can do can also be done in the app and vice versa.-->
```
Comment (erondpowell) [inline docs/getting-started/introduction.md:31]: Need to fact-check this statement. As I feel like the API can do a couple extra things the app can't do. @rijkvanzanten

File: docs/getting-started/introduction.md [hunk 2]
```diff
@@ -1,121 +1,223 @@
...
+
+The platform is guided by the following core principles:
+
+- **Pure** — No predefined or proprietary schema, with all system metadata stored separately.
+- **Open** — Directus Core is public and open source, with no obfuscated or cloud-only code.
+- **Portable** — Database can be exported or migrated at any time with no vendor lock-in.
+- **Limitless** — No artificial limits or paywalls on users, roles, languages,
+  [Collections](/getting-started/collections), or [Items](getting-started/glossary/items).
+- **Extensible** — Every aspect of the platform is modular to avoid any hard feature ceilings.
+- **Unopinionated** — Choose the stack, database, and architecture as you wish.
```
Comment (erondpowell) [inline docs/getting-started/introduction.md:195]: @benhaynes So, I put these here in `Why use Directus`, which I placed right before `Directus Cloud`, the final CTA. I think these concepts fit here, as users can absorb what they mean more easily after some basic concrete details. The guiding principles are also essentially value-offerings. Lastly, the sections we originally talked about, `What's Directus` and `How It Works` are also quite long already!

Let me know if you can dig it or I need to move them out.

File: docs/getting-started/introduction.md [hunk 3]
```diff
@@ -1,121 +1,223 @@
...
+
+**Directus is a simple, data-first solution to complex problems and using it has no downsides.**
+
+Directus was created in 2004, and has been slowly, iteratively improved on for a long time. The core team has carefully
+thought through the app, beginning-to-end, to make sure that using this platform is an all-benefit and no downside
+experience. This platform is plug-and-play, so you're free to link or remove it any time, with no impact on your data.
+You have no vendor lock-in whatsoever, your data is yours. There are no artificial limits on data _(e.g. limits on users
+or roles)_.
+
+<!-- highlight the cost-of-scaling that comes with a platform which charges for users and roles -->
```
Comment (erondpowell) [inline docs/getting-started/introduction.md:179]: @benhaynes Had this idea too, you want me to hit the trigger and add in that argument more explicitly here?

File: docs/getting-started/introduction.md [hunk 4]
```diff
@@ -1,121 +1,223 @@
...
+
+- **Pure** — No predefined or proprietary schema, with all system metadata stored separately.
+- **Open** — Directus Core is public and open source, with no obfuscated or cloud-only code.
+- **Portable** — Database can be exported or migrated at any time with no vendor lock-in.
+- **Limitless** — No artificial limits or paywalls on users, roles, languages,
+  [Collections](/getting-started/collections), or [Items](getting-started/glossary/items).
+- **Extensible** — Every aspect of the platform is modular to avoid any hard feature ceilings.
+- **Unopinionated** — Choose the stack, database, and architecture as you wish.
+
+<!-- No migration- Directus Cloud?? -->
```
Comment (erondpowell) [inline docs/getting-started/introduction.md:197]: @benhaynes you asked me to highlight "no migration" as a thing in "Why" section,  but I felt weird working it in, as we are CTA'ing cloud, which doesn't have BYODB _(doesn't have it yet that is, haha)_.

You want still want me to work this in somehow? Or let it go for the moment?

File: docs/getting-started/introduction.md [hunk 5]
```diff
@@ -1,121 +1,223 @@
...
+  across websites, apps, kiosks, digital signage... _the sky's the limit!_
+- **Internal Tool Builder**\
+  The whole team can build custom apps! Ditch the spreadsheet and quickly build back-office apps and admin panels for customers,
+  inventory, projects, marketing _or anything else._
+- **Data Management and Analytics**\
+  Establish _a single source of truth_ for all data. Build no-code analytics dashboards to gain insights into company KPIs
+  and other metrics. Coalesce previously siloed department data.
+
+<!--
+mention SaaS in there
```
Comment (erondpowell) [inline docs/getting-started/introduction.md:160]: @benhaynes you listed this as a buzzword to add, tbh I'm not sure what you meant. (I know what SaaS, but idk how you wanted me to work it in... Directus as SaaS, Directus as a way to manage other SaaS, or something other???

**Review by erondpowell — COMMENTED** [review]
OK. good to go.

**Comment by erondpowell** [comment]
@benhaynes @rijkvanzanten

This Introduction doc is looking pretty decent, in my opinion. 
If you guys think it's _ok_, I suggest we merge it, then double back later to perfect the copy/structure.

File: docs/getting-started/resources.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+- **[GitHub](https://github.com/directus/directus)** — The open-source repository and version control.
+- **[NPM Package](https://www.npmjs.com/package/directus)** — The official Directus node package.
+- **[Docker Image](https://hub.docker.com/r/directus/directus)** — The official Directus docker image.
+- **[Documentation](https://docs.directus.io)** — Docs for the most recent version of Directus.
+- **[Crowdin](https://locales.directus.io/)** — Service for managing the App's many language translations.
+
+### Community
+
+- **[Youtube](https://www.youtube.com/c/DirectusVideos)** — Checkout our channel with video tutorials and feature
```
Comment (rijkvanzanten) [inline docs/getting-started/resources.md:20]:
```suggestion
- **[YouTube](https://www.youtube.com/c/DirectusVideos)** — Checkout our channel with video tutorials and feature
```

File: docs/getting-started/introduction.md [hunk 7]
```diff
@@ -1,121 +1,216 @@
...
+database clients (like _phpMyAdmin_) work. However, Database Introspection has many unique advantages:
+
+- Absolute control over your pure SQL database schema.
+- Complete transparency, portability, and security for your data.
+- Allows importing existing databases, unaltered and without migrations.
+- Direct database access and the full power of raw, complex SQL queries.
+- Significant performance improvements through optimizations and indexing.
+
+In contrast, other platforms typically use a predefined or proprietary _one-size-fits-all_ data model to store content
+_(have you ever peeked behind the curtain of a WordPress installation? Yikes!)_. That is not the case with Directus.
```
Comment (rijkvanzanten) [inline docs/getting-started/introduction.md:79]:
```suggestion
. That is not the case with Directus.
```

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**